CONTRIBUTING.md

```diff
@@ -0,0 +1,345 @@
+
+### The `_modulename/` directory
+
+Any files that are relevant for the module should be placed in the `_modulename/` directory.
+This includes images that are linked in the `modulename.md` file, as well as slides, and images used in the slides.
+
+When embedding images in the `modulename.md` file or slides, please use the following path: `stories/_modulename/image.png`.
+While technically possible, because we want the modules to be transportable, please avoid using images from other modules.
+
+### The `modulename.md` file
+
+#### YAML header
+
+The module starts with a YAML header.
+For instance, the header for the SMP module looks like this:
+
+```yaml
+---
+id: 1
+trl: medium
+category: Reusability
+title: Software Management Plans
+author: eScience Center
+thumbnail: "nlesc-dummy.png"
+---
+```
+
+The only important fields at this point are `category`, `title`, and `thumbnail`.
+
+The `category` field should be one of the categories mentioned on the [main page](https://esciencecenter-digital-skills.github.io/software-support-essentials/).
+Any changes to the order of categories or their names should be set in the config.json file, in the root directory.
+The `title` field should be the title of the module; it will be shown on the main overview page.
+The `thumbnail` field should be the name of the thumbnail image, which should be placed in the `static/stories/_modulename/` directory.
+This image shows up on the index page.
+Eventually, we will use customized icons for this; currently, we use dummy images.
+Check any of the existing modules for an example.
+
+The `id`, `author` and `trl` (technical readiness level) properties are currently not used, but they are still here for legacy reasons.
+
+Note that the path to the thumbnail image is relative to the `_modulename/` directory, as opposed to any images inside the text.
+
+#### Chapters
+
+A chapter (i.e. the content in a single tab) is fenced by triple colons, as follows:
+
+```markdown
+:::Chapter{headline="Why an SMP?" widemd=1}
+
+A software management plan (SMP) can help you!
+:::
+```
+
+The `headline` field is the title of the of the chapter tab.
+The `widemd` field should be set to 1; this indicates that the chapter is text-based.
+
+Between the colons, you can use markdown to format the text.
+
+#### Chapter types
+
+A standard chapter header looks like this:
+
+```markdown
+:::Chapter{headline="Tab text" CHAPTERTYPE}
+```
+
+Instead of `CHAPTERTYPE`, the following code can be used to refer to different types of content in the chapter:
+
+- A text-based chapter: `widemd=1`
+- A presentation chapter, which embeds slides: `presentation="path/to/slides.md"`, where the path is relative to `_modulename/`
+- An embedded website: `website="https://esciencecenter.nl/"`
+- A YouTube video: `video="Ps8jOj7diA0"`, where the ID is the part after `v=` in the YouTube URL
+- A large image: `image="multimedia_image.avif"`, where the path is relative to `_modulename/`
+
```

Comment:
This is a good suggestion; can we turn it into an issue?

Currently, there are no fully complete modules, so it feels a bit off to refer to one directly at the moment. I do want to refer to them as soon as they are complete, because it is a good point: an example always works best.